packages/frontend/src/creation-form/i18n/en.json

```diff
@@ -1,8 +1,14 @@
 {
+    "info": "Need more info on how Reality.eth works? Check out the documentation page ",
+    "info.here": "here",
     "label.arbitrator": "Arbitrator:",
+    "info.arbitrator": "Anyone who wants to challenge the final answer but doesn't have enough funds to post it directly can request for an arbitrator to step in. Keep in mind the arbitrator can assign a cost for this service.",
     "label.reality.template": "Result type:",
+    "info.reality.template": "The  answer can be either yes or no or a specific number value depending on your needs.",
     "label.question": "Question:",
+    "info.question": "What's the question that the oracle should answer?",
     "label.question.timeout": "Question timeout:",
+    "info.question.timeout": "The timeout period dictates for how long a posted answer can be overridden by another answer with a doubled bond. Each new answer resets the the countdown period. The final unchallenged answer after the countdown period expires is the winner.",
```

Comment:
Double `the the countdown`

Reply:
Addressed in 7900adc9180edc26915bab93ad3762c7b31abf7c